pages/colors/colors.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href='../../index.css'>
+  <link rel="stylesheet" href="./colors.css">
+  <title>starUI🌟 | Colors</title>
+</head>
+<body>
+  <div class="body-center">
+    <nav class='navbar'>
+      <div class="logo">
+        <h2>starUI🌟</h2>
+      </div>
+      <div class="right-nav">
+        <ul>
+          <li>Docs</li>
+          <li>Github</li>
+          <li>Twitter</li>
+        </ul>
+      </div>
+    </nav>
+    <main class="container-flex">
+      <aside class="aside">
+        <h4 class='underline'>Get Started</h4>
+        <ul>
+          <li><a href="../introduction/introduction.html">Introduction</a></li>
+          <li><a href="./colors.html">Colors</a></li>
+          <li><a>Typography</a></li>
+          <li><a>Conclusion</a></li>
+        </ul>
+        <h4 class="underline">Components</h4>
+        <ul>
+          <li><a>Avatar</a></li>
+          <li><a>Alert</a></li>
+          <li><a>Badge</a></li>
+          <li><a>Button</a></li>
+          <li><a>Card</a></li>
+          <li><a>Image</a></li>
+          <li><a>Input</a></li>
+        </ul>
+      </aside>
+      <section class="section">
+        <article class="section-content underline">
+          <h2>starUI's Color Scheme</h2>
+          <p class="small-text">starUI has pre-defined color pallete which is compatible with all it's components. The colors are used in a generic basis which can be accessed using pre-defined classnames.</p>
+        </article>
+        <article class="section-content underline">
+          <h2>Default Palette</h2>
+          <p class="small-text">Below is the color palette of starUI along with the hex code.</p>
+          <div class="color-example">
+            <div class='primary'>
```

Comment:
**FIX**: You can bring these colors in the color palette in a single line. It looks a bit off having 2 colors below and 3 above

![Screenshot 2022-01-27 at 21 23 44](https://user-images.githubusercontent.com/62737267/151394638-016178d5-d18d-454d-964d-ac96b01aa75f.png)
.